ydb/core/tx/schemeshard/schemeshard__monitoring.cpp

```diff
@@ -786,7 +786,6 @@ struct TSchemeShard::TTxMonitoring : public NTabletFlatExecutor::TTransactionBas
                             TABLEH() {str << "DebugMessage";}
                             TABLEH() {str << "SeqNo";}
                             TABLEH() {str << "Processed";}
-                            TABLEH() {str << "Billed";}
```

Comment:
maybe print the number of rows and bytes read/written here?

Reply:
Billed was removed from per shard status some time ago.
I just noticed that it wasn't removed from this line